Pages/Crime.cs

```diff
@@ -10,7 +10,7 @@ namespace QuickTypeCrimes
 {
     using System;
     using System.Collections.Generic;
-
+    using System.ComponentModel.DataAnnotations;
```

Comment:
Added "[DateTime(DateType.Date)]" syntax for displaying the Date and Time and for that I added "ComponentMode1.DateAnnotations" in Index.cshtml